book/Mars_Maps.md

```diff
@@ -0,0 +1,3 @@
+# Making some Mars maps with PyGMT
+
+TODO
```

Comment:
Just a reminder to remove this file when you have time

Reply:
Yep, done in 414c38c953e15e7e2ed4cf74a0e735141f3bcb1b!